k8s/crd-samples/replica-resource.yaml

```diff
@@ -1,9 +1,12 @@
 apiVersion: openebs.io/v1alpha1
-kind: CstorReplica
+kind: CStorVolumeReplica
 metadata:
   name: pvc-ee171da3-07d5-11e8-a5be-42010a8001be-cstor-rep-9440ab
-  poolguid: 7b99e406-1260-11e8-aa43-00505684eb2e
+  annotations:
+    poolGUID: 7b99e406-1260-11e8-aa43-00505684eb2e
```

Comment:
can this be "openebs.io/cstor-pool-guid" ? Or cStorPoolGUID?

Reply:
okay, will use openebs.io/cstor-pool-guid